resources/teamcity/developer/developer-actions.inc.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+developer_install_dependencies_on_linux_action() {
```

Comment:
This says `_on_linux_` but also makes changes to mac. Probably moot because I am asking to roll the mac changes back.

Reply:
Wait - what? I thought I removed the macOS part! I'll have to check how this ended up in here again...

Reply:
Phew, that's actually a relief, was worried we had had some very bad communication!

Reply:
Looks like I made the changes but then forgot to save the file :disappointed: